app/controllers/searches_controller.rb

```diff
@@ -159,6 +159,7 @@ def search_consumer_urls
   end
 
   def docs_search_klass
+    @search_options[:affiliate].search_engine = 'SearchGov' if @search_options[:document_collection]&.too_deep_for_bing?
     return I14ySearch if gets_i14y_results?
```

Comment:
Shouldn't this be:
```ruby
def docs_search_klass
  return I14ySearch if gets_i14y_results? || @search_options[:document_collection]&.too_deep_for_bing?
```
?

Reply:
The search engine needed to be set in addition to sending I14ySearch as the search class.